sampleSpecs/protoTest.smithy

```diff
@@ -197,15 +177,85 @@ structure InlinedUnionWrapper {
 
 @protoInlinedOneOf
 union MyInlinedUnion {
-    @protoIndex(1)
     int: Integer
-    @protoIndex(2)
     bool: Boolean
 }
 
 @protoEnabled
 structure StringMapWrapper {
-    @protoIndex(1)
     @required
     values: StringMap
 }
+
+map StringMap {
+    key: String
+    value: Integer
+}
+
+@protoEnabled
+structure Enums {
+    @required
+    closedString: ClosedString
+    @required
+    openString: OpenString
+    @required
+    closedInt: ClosedInt
+    @required
+    openInt: OpenInt
+}
+
+// NB : order of enum values generated by smithy4s is not
```

Comment:
**question:** is not what?

Reply:
This what meant to say that the ordering of enum values is not guaranteed. I've removed the comment in favour of this PR : https://github.com/disneystreaming/smithy4s/pull/1437